src/pages/overview/pdf-accessibility-auto-tag-api/quickstarts/dotnet/index.md

```diff
@@ -0,0 +1,333 @@
+---
+title: .NET | Quickstarts | PDF Accessibility Auto Tag API | Adobe PDF Services
+---
+
+# Quickstart for PDF Accessibility Auto Tag API (.NET)
+
+To get started using Adobe PDF Accessibility Auto Tag API, let's walk through a simple scenario - taking an input PDF document and running PDF Accessibility Auto Tag API against it. Once the PDF has been tagged, we'll provide the document with tags and optionally, a report file. In this guide, we will walk you through the complete process for creating a program that will accomplish this task. 
+
+## Prerequisites
+
+To complete this guide, you will need:
+
+* [.NET Core: version 2.1 or above](https://dotnet.microsoft.com/en-us/download)
+* [.Net SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
+* A build tool: Either Visual Studio or .NET Core CLI.
+* An Adobe ID. If you do not have one, the credential setup will walk you through creating one.
+* A way to edit code. No specific editor is required for this guide.
+  
+## Step One: Getting credentials
+
+1) To begin, open your browser to <https://documentservices.adobe.com/dc-integration-creation-app-cdn/main.html?api=pdf-extract-api>. If you are not already logged in to Adobe.com, you will need to sign in or create a new user. Using a personal email account is recommend and not a federated ID.

```

Comment:
This URL should direct to Autotag API credentials page instead of Extract API creds page